app/src/main/java/com/craxiom/networksurveyplus/messages/QcdmUmtsParser.java

```diff
@@ -18,6 +34,7 @@ private QcdmUmtsParser()
     {
     }
 
+    private static final String UMTS_NAS_MESSAGE = "UmtsNas";
```

Comment:
Consistency is important. I am not always the best at this, but I try. Here you used `*_MESSAGE`, and for the WCDMA parser you used `*_MESSAGE_TYPE`. Update this one to use `*_MESSAGE_TYPE` for the name as well.

Reply:
Agreed, updated.